pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -464,8 +464,13 @@ func (r *ReconcileLogStorage) generateSecrets(
 	for _, certName := range certs {
 		cert, err := cm.GetCertificate(r.client, certName, common.OperatorNamespace())
 		if err != nil {
-			r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get certificate", err, log)
-			return nil, err
+			if certificatemanager.IsCertUsageError(err) {
+				msg := fmt.Sprintf("skipping tigera-operator/%s secret it will be added when it is updated: %s", certName, err)
```

Comment:
We shouldn't hard-code the namespace in the log - we have `common.OperatorNamespace()`